po/fr.po

```diff
@@ -681,60 +679,55 @@ msgid "Preferences"
 msgstr "Préférences"
 
 msgid "Installation options:"
-msgstr ""
+msgstr "Options d'installation :"
 
 msgid "Try to remove libraries before (re)installing them?"
-msgstr ""
+msgstr "Essayer de supprimer les bibliothèques avant de les (ré)installer ?"
 
 msgid "Show README of newly installed libraries (if present)?"
-msgstr ""
+msgstr "Afficher le README (si présent) des nouvelles bibliothèques ?"
```

Comment:
Suggesting : "Afficher le README (si présent) des nouvelles bibliothèques installées ?"

Reply:
I preferred "des bibliothèques nouvellement installées".

Reply:
You're right.